Test/Core/InitDualityAttribute.cs

```diff
@@ -25,21 +26,18 @@ public void BeforeTest(ITest details)
 
 			// Set environment directory to Duality binary directory
 			this.oldEnvDir = Environment.CurrentDirectory;
-			string codeBaseURI = typeof(DualityApp).Assembly.CodeBase;
-			string codeBasePath = codeBaseURI.StartsWith("file:") ? codeBaseURI.Remove(0, "file:".Length) : codeBaseURI;
-			codeBasePath = codeBasePath.TrimStart('/');
-			Console.WriteLine("Testing Core Assembly: {0}", codeBasePath);
-			Environment.CurrentDirectory = Path.GetDirectoryName(codeBasePath);
+			Console.WriteLine("Testing Core Assembly: {0}", TestContext.CurrentContext.TestDirectory);
+			Environment.CurrentDirectory = TestContext.CurrentContext.TestDirectory;
```

Comment:
The new code is only equivalent to the old one when the testing binary is in the same folder as `Duality.dll`, which formerly was `Build/Output` in both cases. I don't currently have the full overview with all the recent changes, but if this is still true this should work out.

Reply:
However, the console log is no longer correct: The printed path is actually the directory, not the assembly (which is fine, but not as advertised). Same for the console log in the editor part down below. Maybe just change the line to something like "Testing in working directory: '{0}'"?